clients/csharp-uwp/UWPVoiceAssistantSample/App.xaml.cs

```diff
@@ -43,11 +43,11 @@ public App()
             this.logger = LogRouter.GetClassLogger();
             this.logger.Log("Constructor: app launched");
 
+            Task.Run(async () => await LocalSettingsHelper.InitializeAsync()).Wait();

```

Comment:
Not in love with this but out of scope for immediate changes

Reply:
Ah- we added the null check in keyword registration to avoid this. This is going to increase time from activation to showing UI. Why can't we do this async?